ucsb-cs156-f24/proj-happycows-f24-10
# Add pageSize parameter to OurTable

## Overview

Allowed OurTable to have pagesize passed in as a parameter. If not passed it defaults to 10 (old behavior). Also added a two storybook entries for pagination. I know the screenshots look weird because the nav bar is right aligned, but that was just the old behavior.

## Screenshots (Optional)

<img width="713" alt="Screenshot 2024-11-21 at 6 07 53 PM" src="https://github.com/user-attachments/assets/33dab353-823b-4daa-93de-a2167ee7db03">
<img width="766" alt="Screenshot 2024-11-21 at 6 07 41 PM" src="https://github.com/user-attachments/assets/ff488aaa-d689-41f9-b828-499387525778">

## Future Possibilities (Optional)

Some possible points could be:
- We should probably fix the alignment of the nav bar.
- When we finish the OurTable test overhaul, we should add tests for this.

## Validation (Optional)

I can't get the storybook on github to work, so for now you need to:
 1. download the branch locally
 2. run `npm run storybook` in `/frontend`
 3. check the OurTable component stories

## Tests


## Linked Issues

Closes #48
